.github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATES/pull_request_template.md

```diff
@@ -0,0 +1,56 @@
+   <!-- If you need help with you PR or if you want to to discuss something with us, join the WalkingPal [Discord community](https://discord.gg/Et3aQTsW6a)! -->
```

Comment:
The folder name is wrong. Its PULL...MPLATE. (no S)

Reply:
This PR TEMPLATE has unnecssary lines. Delete everything and put only the lines as given in this file: https://github.com/WalkingPal/walking-pal-web/pull/77/files